playbooks/intel_ocp_provisioning.yaml

```diff
@@ -0,0 +1,90 @@
+- hosts: localhost
+  gather_facts: no
+  vars:
+    kubeconfig_path: "~/.kube/config"
+  environment:
+    KUBECONFIG: "{{ kubeconfig_path }}"
+  vars_prompt:
+    - name: "install_operators"
+      prompt: "Do you want to install operators? 'Yes' to install NFD Operator and Intel Device Plugins Operator, or 'No' to skip"
+      private: no
+    - name: "validation_feature"
```

Comment:
the playbook here is just for Automation testing? or it will also be used by the oneClick solution?

Reply:
@uMartinXu   One-click is an extension of playbooks where we added whole e2e in one playbook.. If we can decouple different modules then we can use same playbooks for one-click solution too..